Mesh_2/doc/Mesh_2/Mesh_2.txt

```diff
@@ -278,6 +278,7 @@ After Delaunay refinement, angles are in the interval [28.5; 121.9] degrees.
 After 10 iterations of Lloyd optimization, they are in [29.1; 110.8]. 100 iterations take them to [29.3; 109.9].
 \cgalFigureEnd
 
+Since \cgal 5.6, `lloyd_optimize_mesh_2()` uses \ref bgl_namedparameters to set parameters. More details are provided in  \ref FromBoostNPtoCGALNP.
```

Comment:
it seems to me that "since" + "uses" (present) is not correct. I'd better be "Since 5.6, lloyd has used" or "has been using"

Reply:
Let's change `Since \cgal 5.6` to `As of \cgal 5.6`, because that usage allows the present.